### PR TITLE
remove dependency on ping/ping6 command, send icmp packet natively

### PR DESCRIPTION
**Pull request by l-hellmann**
remove ping/ping6 dependency

### CODE REVIEW

File: src/i18n/en.go [hunk 1]
```diff
@@ -221,7 +221,7 @@ const (
 	VpnStatusDnsStatusUnset          = "dns isn't set, try `/path/to/zcli vpn start` command"
 	VpnStatusAdditionalInfo          = "additional info:"
 	VpnStatusDnsCheckError           = "we failed to check that dns is working correctly"
-	VpnStatusDnsNoCheckFunction      = "there is no function for dns check"
+	VpnStatusCheckInvalidAddress     = "invalid address"
```
Comment (kawaiiyoshiyuki) [inline src/i18n/en.go:224]: as a user I'd like to know what address you talk about, e.g. "invalid IPv6 / IPv4 address"

File: src/i18n/en.go [hunk 2]
```diff
@@ -273,6 +273,7 @@ more info: https://docs.zerops.io/documentation/cli/authorization.html`
 	GrpcVpnApiTimeout = "zerops vpn server didn't respond within assigned time, try it again later"
 
 	HintChangeRegion = "hint: try to change your region (you can list available regions using `zcli region list`)"
+	ContextDeadline  = "context should have deadline"
```
Comment (kawaiiyoshiyuki) [inline src/i18n/en.go:276]: as a user I have absolutely no idea what that means or what to do, can we suggest or describe it more precisely?

File: src/nettools/ping.go [hunk 3]
```diff
@@ -2,20 +2,108 @@ package nettools
...
+
+	b, err := m.Marshal(nil)
+	if err != nil {
+		return err
+	}
+
+	n, err := c.WriteTo(b, dst)
+	if err != nil {
+		return err
+	} else if n != len(b) {
```
Comment (kawaiiyoshiyuki) [inline src/nettools/ping.go:48]: do we need the `else if` ? it looks like the `if` is sufficient

**Review by kawaiiyoshiyuki — COMMENTED** [review]
CHANGELOG is mandatory, pls update it

File: src/i18n/en.go (hunk 2 above)
Comment (l-hellmann) [reply src/i18n/en.go:276]: this is more like error for dev, ping function expects context to has deadline 
```
	deadline, has := ctx.Deadline()
	if !has {
		return errors.New(i18n.ContextDeadline)
	}
```
any for better solution?

File: src/nettools/ping.go (hunk 3 above)
Comment (l-hellmann) [reply src/nettools/ping.go]: Yes, we check if `WriteTo` returned error and if not, that count of bytes written is equal to length of buffer.

**Comment by l-hellmann** [comment]
> CHANGELOG is mandatory, pls update it

Updated, check if its sufficient

File: src/i18n/en.go (hunk 2 above)
Comment (vnarek) [reply src/i18n/en.go:276]: I would return internal server error or something like that

**Pull request closed without merging**